build-artifacts/project-template-gradle/settings.gradle

```diff
@@ -5,3 +5,74 @@ include ':app'//, ':runtime', ':runtime-binding-generator'
 //project(':runtime-binding-generator').projectDir = new File("${System.env.ANDROID_RUNTIME_HOME}/test-app/runtime-binding-generator")
 
 file("google-services.json").renameTo(file("./app/google-services.json"))
+
+import org.gradle.internal.logging.text.StyledTextOutputFactory
+
+import java.nio.file.Paths
+
+import org.gradle.internal.logging.text.StyledTextOutputFactory
+import groovy.json.JsonSlurper
+import static org.gradle.internal.logging.text.StyledTextOutput.Style
+
+def USER_PROJECT_ROOT = "$rootDir/../../"
+def outLogger = services.get(StyledTextOutputFactory).create("colouredOutputLogger")
+def ext  = {
+    appResourcesPath = getProperty("appResourcesPath")
+    appPath = getProperty("appPath")
+}
+
+def getAppPath = { ->
+      def relativePathToApp = "app"
```

Comment:
Isn’t this a user-customisable path? Not sure it’s guaranteed to be called “app”.

Reply:
I think it's the same everywhere, the above function is copied as is from the build.gradle file. https://github.com/ammarahm-ed/android-runtime/blob/3b4a56df86f36a7738cc0089195d20239ce67273/test-app/build.gradle#L59